libuuu/hidreport.cpp

```diff
@@ -92,6 +92,11 @@ int HIDReport::write(const void *p, size_t sz, uint8_t report_id)
 			notify(off, uuu_notify::NOTIFY_TRANS_POS);
 		}
 	}
-	notify(off, uuu_notify::NOTIFY_TRANS_POS);
+
```

Comment:
I think directly use sz. 

notify(sz, uuu_notify::NOTIFY_TRANS_POS);

Reply:
Yeah! that's right.

Reply:
Can you merge two commits to one?